README.md

```diff
@@ -15,6 +15,15 @@ If you are still on `v1` make sure to follow [migration guide](https://vue-query
 
 If you are already on `v2`, just swap `vue-query` for `@tanstack/vue-query` in both of your `package.json` and `import` statements. Everything should still work as before.
 
+steps to move to `@tanstack/vue-query` from `vue-query` v2
+
+1. delete your `node_modules`
+2. delete your `lock` file
+3. remove `vue-query` from your package-json
+4. install deps
+5. install `@tanstack/vue-query`
```

Comment:
Should be merged with step 3 (or right after). Like this:

3\. Replace `"vue-query": "2.x"` with `"@tanStack/vue-query": "4.19.x"` in package.json

4\. Run `npm install`